### PR TITLE
Change file suffix .js to .jsx

### PR DESCRIPTION
**Pull request by tacigar**
Maybe it is widely used naming convention.
This is a trivial PR, but I have been concerned about this.

### CODE REVIEW

File: zipkin-lens/src/components/App/Sidebar/index.js [hunk 1]
```diff
@@ -14,86 +14,4 @@
...
-          </a>
-        </div>
-      </div>
-    );
-  }
-}
-
-Sidebar.propTypes = propTypes;
-
-export default withRouter(Sidebar);
```
Comment (tacigar) [inline zipkin-lens/src/components/App/Sidebar/index.js]: Move the above codes to `Sidebar.jsx`

File: zipkin-lens/src/components/App/index.js [hunk 2]
```diff
@@ -14,54 +14,4 @@
...
-              render={TraceViewerContainer}
-            />
-          </Layout>
-        </BrowserRouter>
-      </Provider>
-    );
-  }
-}
-
-export default App;
```
Comment (tacigar) [inline zipkin-lens/src/components/App/index.js:67]: The above code is moved to `App.jsx`

File: zipkin-lens/src/components/App/App.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+              render={TraceViewerContainer}
+            />
+          </Layout>
+        </BrowserRouter>
+      </Provider>
+    );
+  }
+}
+
+export default App;
```
Comment (tacigar) [inline zipkin-lens/src/components/App/App.jsx:67]: The above code is moved from `index.js`

File: zipkin-lens/src/components/App/Sidebar/Sidebar.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          </a>
+        </div>
+      </div>
+    );
+  }
+}
+
+Sidebar.propTypes = propTypes;
+
+export default withRouter(Sidebar);
```
Comment (tacigar) [inline zipkin-lens/src/components/App/Sidebar/Sidebar.jsx]: The above code is moved from `index.js`

File: zipkin-lens/src/components/Browser/Browser.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+          tracesMap={tracesMap}
+        />
+      </div>
+    );
+  }
+}
+
+Browser.propTypes = propTypes;
+
+export default Browser;
```
Comment (tacigar) [inline zipkin-lens/src/components/Browser/Browser.jsx]: The above code is moved from index.js

File: zipkin-lens/src/components/Browser/TraceSummary/TraceSummary.jsx [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+            : null
+        }
+      </div>
+    );
+  }
+}
+
+TraceSummary.propTypes = propTypes;
+
+export default TraceSummary;
```
Comment (tacigar) [inline zipkin-lens/src/components/Browser/TraceSummary/TraceSummary.jsx]: The above code is moved from index.js

File: zipkin-lens/src/components/Browser/TraceSummary/index.js [hunk 7]
```diff
@@ -14,105 +14,4 @@
...
-            : null
-        }
-      </div>
-    );
-  }
-}
-
-TraceSummary.propTypes = propTypes;
-
-export default TraceSummary;
```
Comment (tacigar) [inline zipkin-lens/src/components/Browser/TraceSummary/index.js]: The above code is moved to `TraceSummary.jsx`

File: zipkin-lens/src/components/Browser/index.js [hunk 8]
```diff
@@ -14,59 +14,4 @@
...
-          tracesMap={tracesMap}
-        />
-      </div>
-    );
-  }
-}
-
-Browser.propTypes = propTypes;
-
-export default Browser;
```
Comment (tacigar) [inline zipkin-lens/src/components/Browser/index.js]: The above code is moved to `Browser.jsx`

File: zipkin-lens/src/components/Dependencies/Dependencies.jsx [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+Dependencies.propTypes = propTypes;
+
+export default withRouter(Dependencies);
```
Comment (tacigar) [inline zipkin-lens/src/components/Dependencies/Dependencies.jsx]: The above code is moved from `index.js`

File: zipkin-lens/src/components/Dependencies/index.js [hunk 10]
```diff
@@ -14,215 +14,4 @@
...
-          />
-        </div>
-      </div>
-    );
-  }
-}
-
-Dependencies.propTypes = propTypes;
-
-export default withRouter(Dependencies);
```
Comment (tacigar) [inline zipkin-lens/src/components/Dependencies/index.js]: The above code is moved to `Dependencies.jsx`

File: zipkin-lens/src/components/MiniTimeline/MiniTimeline.jsx [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+        endTs={endTs}
+        onStartAndEndTsChange={onStartAndEndTsChange}
+      />
+    </div>
+  );
+};
+
+MiniTimeline.propTypes = propTypes;
+
+export default MiniTimeline;
```
Comment (tacigar) [inline zipkin-lens/src/components/MiniTimeline/MiniTimeline.jsx]: The above code is moved from `index.js`

File: zipkin-lens/src/components/MiniTimeline/index.js [hunk 12]
```diff
@@ -14,55 +14,4 @@
...
-        endTs={endTs}
-        onStartAndEndTsChange={onStartAndEndTsChange}
-      />
-    </div>
-  );
-};
-
-MiniTimeline.propTypes = propTypes;
-
-export default MiniTimeline;
```
Comment (tacigar) [inline zipkin-lens/src/components/MiniTimeline/index.js]: The above code is moved to `MiniTimeline.jsx`

File: zipkin-lens/src/components/App/App.jsx (hunk 3 above)
Comment (tacigar) [reply zipkin-lens/src/components/App/App.jsx:67]: I did same renaming on other files.

**Review by codefromthecrypt — APPROVED** [review]
I ran from this branch and clicked everywhere after loading traces. works fine

**Comment by codefromthecrypt** [comment]
merging immediately as change like this tends to drift annoyingly when not :P

Thanks @tacigar!

**Pull request merged**